src/common/ErrorPage/styled.js

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+
+export const Container = styled.main`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    max-width: 100%;
+`
+
+export const Element = styled.img`
+    margin: 120px 0px 40px;
```

Comment:
Jutro musimy dogadać się co do marginesów - wydaje mi się że najlepiej będzie dodawać je tylko elementom od góry. Jak będą z góry i z dołu elementu to przy składaniu strony w całość może się wszystko porozjeżdżać 😅